src/omnicore/tx.cpp

```diff
@@ -2138,7 +2138,15 @@ int CMPTransaction::logicMath_EnableFreezing()
         return (PKT_ERROR_TOKENS -49);
     }
 
-    enableFreezing(property, block);
+    int liveBlock = 0;
+    if (!IsFeatureActivated(FEATURE_FREEZENOTICE, block)) {
+        liveBlock = block;
+    } else {
+        const CConsensusParams& params = ConsensusParams();
+        liveBlock = params.OMNI_FREEZE_WAIT_PERIOD + block;
```

Comment:
Maybe I'm missing something, but can you quickly clarify what happens, if the sender wants to activate freezing at block height 1000? Does he have to send 1000-waitperiod or 1000?

Reply:
The sender doesn't have a choice in the wait period, it's defined in the consensus rules bud